### DISCUSSION TITLE
mockoon serverless on aws - getting Request body

### DISCUSSION THREAD

**Discussion by indraneel76** [mockoon/mockoon#976] [Help]
Hi , 
I am interested to get the request body logged in cloudwatch which is sent to mockoon as part of POST request. Currently it doesn't show the payload in AWS cloudwatch
Any idea how to get that

**Comment by 255kb** [comment]
Hi, 
The serverless package is a bit basic right now and does not log anything like the CLI. 
This should be added indeed. The relevant code could be extracted from the CLI and shared with this package. 

Could you please open an issue? 
Thanks!
